lib/destinations/tumblr.js

```diff
@@ -185,25 +140,22 @@ class DestinationTumblr {
       const sendData = {
         layout,
         content,
-        tags: tags.join(','),
+        tags,
         state: this.publishState
       };
       console.log('🙋‍♂️ Submitting data');
       console.log(JSON.stringify(sendData, null, 2));
-      const { response } = await this.makeNpfRequestForm(
-        `/blog/${this.blogName}/posts/${this.ask.id}`,
-        formData,
+      const response = await this.client.editPost(
```

Comment:
This was surprising to me, but `PUT` to this endpoint corresponds to `editPost`. I didn't validate this behavior.